drivers/storage/portworx/util/util.go

```diff
@@ -99,8 +99,20 @@ const (
 	// EnvKeyPortworxAuthJwtIssuer is an environment variable defining the PX Security JWT Issuer
 	EnvKeyPortworxAuthJwtIssuer = "PORTWORX_AUTH_JWT_ISSUER"
 
-	// SecurityPXAuthKeysSecretName is the admin secret name for PX security
-	SecurityPXAuthKeysSecretName = "px-auth-keys"
+	// SecurityPXSystemSecretsSecretName is the secret name for PX security system secrets
+	SecurityPXSystemSecretsSecretName = "px-system-secrets"
```

Comment:
Any reason this is plural while the other is not?

Reply:
@piyush-nimbalkar I was thinking this would include the apps issuer secret key as well.

Reply:
Oh right, sounds good then.